connectivity/FEATURE_BLE/source/cordio/source/BLEInstanceBaseImpl.cpp

```diff
@@ -366,17 +375,22 @@ void BLEInstanceBase::stack_handler(wsfEventMask_t event, wsfMsgHdr_t *msg)
             };
 #if BLE_FEATURE_EXTENDED_ADVERTISING
             // initialize extended module if supported
+            tr_info("Reset sequence complete - initialising extended features");
             if (HciGetLeSupFeat() & HCI_LE_SUP_FEAT_LE_EXT_ADV) {
 #if BLE_ROLE_BROADCASTER
+                tr_debug("DmExtAdvInit");
```

Comment:
Thanks for adding these traces, I think it can even be a `tr_info` as it is only print once and it would be very valuable for bug report.

Reply:
upgraded to tr_info